atc/resource/resource_put.go

```diff
@@ -21,9 +22,10 @@ func (resource *resource) Put(
 
 	vr := &VersionResult{}
 
+	path := "/opt/resource/out"
```

Comment:
This could be a `const` instead.

Why is immutable better? Because when as a reviewer I read `const`, I can relax and not worry if somewhere in the code this value is changed.